core/spec/controllers/mno_enterprise/i18n_spec.rb

```diff
@@ -17,24 +17,39 @@ def index
         I18n.default_locale = :en
       end
 
-      context 'when a valid locale is provided' do
-        it 'sets the provided locale' do
-          get :index, locale: :fr
-          expect(I18n.locale).to eq(:fr)
+      context 'when I18n is disabled' do
+        before { MnoEnterprise.i18n_enabled = false }
```

Comment:
define `subject { I18n.locale }` then use `is_expected.to eq(foo)` in your specs.

Reply:
done